lib/Email/MIME.pm

```diff
@@ -119,7 +119,15 @@ my $NO_ENCODE_RE = qr/
 /ix;
 
 sub new {
-  my $self = shift->SUPER::new(@_);
+  my ($class, $text, $arg, @rest) = @_;
+  $arg ||= {};
+
+  my $encode_check = exists $arg->{encode_check}
+                   ? delete $arg->{encode_check}
+                   : 1; # Encode::FB_CROAK
```

Comment:
Email::MIME already uses '1' directly and I assume it either has a very good reason or wasn't aware of FB_CROAK at the time.

If it's the first, I don't want to break anything, so I think leaving it as 1 is fine for now